jekyll/_cci2/vs-code-extension-overview.adoc

```diff
@@ -165,12 +165,51 @@ Both of these commands can be run by:
 
 - Right-clicking anywhere within an opened `.circleci/config.yml` file.
 
-- Clicking on the CircleCI button on the top right of the editor when you have a `.circleci/config.yml`open. Note that the button will not be visible if you are editing any other file.
+- Clicking on the CircleCI button on the top right of the editor when you have a `.circleci/config.yml` open. Note that the button will not be visible if you are editing any other file.
 
 [#open-source-language-server]
 === Open source language server
 
-The config helper is based on a dedicated language server specific for CircleCI YAML files, which is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[circleci-yaml-language-server].
+The config helper is based on a dedicated language server specific for CircleCI YAML files, which is open source. You can view its source code, contribute and add issues directly on the project repository: link:https://github.com/CircleCI-Public/circleci-yaml-language-server[CircleCI YAML language server].
 
 You can also integrate the language server into any editor which supports the Language Server Protocol, and build your own plugin to benefit from config helper capabilities in your favourite editor.
 
+[#test-run-your-config-from-vs-code]
+== Test run your config from VS Code
+
+NOTE: The ability to trigger a pipeline from VS Code with a _custom_ config is currently in **closed preview**. This feature is available on the **Free Plan** for accounts integrated through the **GitHub OAuth app**, or **Bitbucket**. To find out which GitHub account type you have, see the xref:github-integration#[GitHub OAuth app integration] page.
```

Comment:
> for accounts integrated through
maybe "for **projects** integrated through" is better, since it's controlled at the project level not user level.

Reply:
Let's go with `unversioned config` instead of `custom config`.